vtu11/impl/vtu11_impl.hpp

```diff
@@ -146,6 +150,43 @@ void write( const std::string& filename,
   output.close( );
 }
 
+//ParallelWrite generates a pvtu file and accordingly the vtu pieces in a subfolder
+//Each piece consists of a set of points and cells
+template<typename MeshGenerator, typename Writer = AsciiWriter>
+void parallelWrite( const std::string& path,
+                    std::string baseName,
+                    MeshGenerator& mesh,
+                    const std::vector<DataSet>& pointData,
+                    const std::vector<DataSet>& cellData,
+                    size_t fileId, size_t numberOfFiles,
+                    Writer writer = Writer())
+{
+	//ToDo: Take care of cleaning the folder! not done in this code as Kratos takes care of it
+    fs::path p1 = path;
+    p1.make_preferred();
+    if( !fs::exists( p1 ) )
+    {
+      fs::create_directory( p1 );
+      std::cout << "Original path, where the parallel files should be stored, does not exist!" << std::endl;
+    }
+
+    fs::path directory = path + baseName + "/";
+    directory.make_preferred();
```

Comment:
maybe better to use the `/` operator here?
See https://en.cppreference.com/w/cpp/filesystem/path/append

not sure, please check

Reply:
I think that the / operator works with fs::path variables, whereas here the baseName is a std::string...I will double-check if it works for strings too.
Otherwise we would either have to change the baseName into fs::path or just concatenate it with the "+" as we do above.

Reply:
Also we consulted this: https://en.cppreference.com/w/cpp/filesystem/path/make_preferred
That's why we added the `directory.make_preferred( );`

Reply:
I usually convert the `std::string` to a `fs::path` to have access to all the features of `fs::path`
=> then you never need to explicitly specify the separator

(BTW afaik `/` us universal and works on any OS)